src/components/form/Admin.jsx

```diff
@@ -10,9 +10,20 @@ import { STATUSES } from "@/data/admin/Admins";
 
 const Admin = () => {
   const { data: session } = useSession();
+
+  function extractFirstName(str) {
```

Comment:
1. arrow function
2. put this as a util and have 1 function return both the first and last name and use that everywhere